playwright/integrations/unauthenticated-chat.spec.ts

```diff
@@ -794,4 +794,43 @@ test.describe('UnauthenticatedChat @UnauthenticatedChat', () => {
         expect(liveChatContextConversationDetails.conversationId).toBeDefined();
         expect(liveChatContextConversationDetails.canRenderPostChat).toBeDefined();
     });
+
+    test('Verify `useCreateConversation.disable: false` triggers Create Conversation flow', async ({ page }) => {

```

Comment:
`'ChatSDK.initialize() with ChatSDK Configuration "useCreateConversation.disable" set to "true" should perform create conversation call`

Reply:
Modified